cfme/configure/configuration/region_settings.py

```diff
@@ -87,6 +87,17 @@ def delete(self, cancel=False):
         view.browser.handle_alert(cancel=cancel)
         view.flash.assert_no_error()
 
+    def change_name(self, name):
```

Comment:
Optional : I think we should rename this method to "update_tag_name"

Reply:
@sshveta I removed this method as I realized that `update` method doesn't do what it is supposed to do. Now I fixed it and can utilize it in this test as well.